code/game/objects/effects/overlays.dm

```diff
@@ -64,6 +64,10 @@
 	icon_state = "laser_target3"
 	layer = ABOVE_FLY_LAYER
 
+/obj/effect/overlay/blinking_laser/Initialize(mapload)
+	. = ..()
+	notify_ai_hazard()
```

Comment:
increase range for this?

Reply:
put it on the wrong thing, meant for the hazard lines not the middle laze bit, updated now